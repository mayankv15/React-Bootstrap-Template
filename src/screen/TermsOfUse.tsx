import React from 'react';

const TermsOfUse = () => {
    return (
        <div className="content-body-div" style={{ display: "block", padding: "50px 150px" }}>
            <div className="inner-page">
                <p style={{ fontSize: "48px", color: "#012653" }}>Terms of use</p>
                <p>The term "User" shall mean the User who is browsing the website www.techsophy.com (Site).</p>
                <p>The domain address www.techsophy.com is owned by Techsophy. All content present on this Site, including all the software, text, images, graphics, video and audio used on this Site, is exclusive and proprietary material owned by Techsophy.</p>
                <p>The Techsophy name and logo are trademarks which belong to and are owned by Techsophy. These trademarks may not be used in any manner whatsoever without prior written consent from Techsophy.</p>
                <p>By using the Site, the User agrees to follow and be bound by the following terms and conditions and these terms and conditions may be revised or modified at any time by Techsophy without any notice to the User:</p>
                <p>No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.</p>
                <p>No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.</p>
                <p>Unauthorized use of the materials appearing on this Site may violate copyright, trademark and other applicable laws, and could result in criminal or civil penalties. Techsophy may monitor access to the Site.</p>
                <p>Techsophy may terminate the User's access to the Site at any time for any reason. The provisions regarding disclaimer of warranty, accuracy of information, and indemnification shall survive such termination.</p>
                <p>Techsophy makes no warranties, express or implied, including, without limitation, those of merchantability and fitness for a particular purpose, with respect to any information, data, statements or products made available on the Site.</p>
                <p>The Site, and all content, materials, information, software, products and services provided on the Site, are provided on an "as is" and "as available" basis.</p>
                <p>Techsophy shall have no responsibility for any damage to User's computer system or loss of data that results from the download of any content, materials, information from the Site.</p>
                <p>Techsophy may unilaterally change or discontinue any aspect of the Site at any time, including, its content or features.</p>
                <p>Techsophy will not be liable for damages of any kind, including without limitation, direct, indirect, incidental or consequential damages (including, but not limited to, damages for lost profits, business interruption and loss of programs or information) arising out of the use of or inability to use the Site, or any information provided on the Site, or for any other reason whatsoever.</p>
                <p>The User agrees to indemnify, defend and hold Techsophy harmless from and against all losses, expenses, damages and costs, including reasonable attorneys' fees, arising out of or relating to any misuse by the User of the content and services provided on the Site.</p>
                <p>The information contained in the Site has been obtained from sources believed to be reliable. Techsophy disclaims all warranties as to the accuracy, completeness or adequacy of such information.</p>
                <p>The Site provides links to web sites and access to content, products and services from third parties, including users, advertisers, affiliates and sponsors of the Site. The User agrees that Techsophy is not responsible for the availability of, and content provided on, third party web sites. The User is requested to peruse the policies posted by other web sites regarding privacy and other topics before use. Techsophy is not responsible for third party content accessible through the Site, including opinions, advice, statements and advertisements, and the User shall bear all risks associated with the use of such content. </p>
                <p>Techsophy is not responsible for any loss or damage of any sort that the User may incur from dealing with any third party.</p>
                <p>Techsophy makes no warranty that: (a) the Site will meet your requirements; (b) the Site will be available on an uninterrupted, timely, secure, or error-free basis; (c) the results that may be obtained from the use of the Site or any services offered through the Site will be accurate or reliable.</p>
            </div>
        </div>
    );
}

export default TermsOfUse;