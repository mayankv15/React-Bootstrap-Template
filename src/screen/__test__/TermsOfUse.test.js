import React from "react";
import { render } from "@testing-library/react";
import TermsOfUse from "../TermsOfUse"

describe("Renders TermsOfUse", () => {
    it("should render TermsOfUse correctly", () => {
        const component = render(<TermsOfUse />);
        expect(component).toMatchSnapshot();
    });
});
