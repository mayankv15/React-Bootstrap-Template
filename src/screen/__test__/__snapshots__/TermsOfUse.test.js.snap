// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders TermsOfUse should render TermsOfUse correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="content-body-div"
        style="display: block; padding: 50px 150px;"
      >
        <div
          class="inner-page"
        >
          <p
            style="font-size: 48px; color: rgb(1, 38, 83);"
          >
            Terms of use
          </p>
          <p>
            The term "User" shall mean the User who is browsing the website www.techsophy.com (Site).
          </p>
          <p>
            The domain address www.techsophy.com is owned by Techsophy. All content present on this Site, including all the software, text, images, graphics, video and audio used on this Site, is exclusive and proprietary material owned by Techsophy.
          </p>
          <p>
            The Techsophy name and logo are trademarks which belong to and are owned by Techsophy. These trademarks may not be used in any manner whatsoever without prior written consent from Techsophy.
          </p>
          <p>
            By using the Site, the User agrees to follow and be bound by the following terms and conditions and these terms and conditions may be revised or modified at any time by Techsophy without any notice to the User:
          </p>
          <p>
            No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.
          </p>
          <p>
            No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.
          </p>
          <p>
            Unauthorized use of the materials appearing on this Site may violate copyright, trademark and other applicable laws, and could result in criminal or civil penalties. Techsophy may monitor access to the Site.
          </p>
          <p>
            Techsophy may terminate the User's access to the Site at any time for any reason. The provisions regarding disclaimer of warranty, accuracy of information, and indemnification shall survive such termination.
          </p>
          <p>
            Techsophy makes no warranties, express or implied, including, without limitation, those of merchantability and fitness for a particular purpose, with respect to any information, data, statements or products made available on the Site.
          </p>
          <p>
            The Site, and all content, materials, information, software, products and services provided on the Site, are provided on an "as is" and "as available" basis.
          </p>
          <p>
            Techsophy shall have no responsibility for any damage to User's computer system or loss of data that results from the download of any content, materials, information from the Site.
          </p>
          <p>
            Techsophy may unilaterally change or discontinue any aspect of the Site at any time, including, its content or features.
          </p>
          <p>
            Techsophy will not be liable for damages of any kind, including without limitation, direct, indirect, incidental or consequential damages (including, but not limited to, damages for lost profits, business interruption and loss of programs or information) arising out of the use of or inability to use the Site, or any information provided on the Site, or for any other reason whatsoever.
          </p>
          <p>
            The User agrees to indemnify, defend and hold Techsophy harmless from and against all losses, expenses, damages and costs, including reasonable attorneys' fees, arising out of or relating to any misuse by the User of the content and services provided on the Site.
          </p>
          <p>
            The information contained in the Site has been obtained from sources believed to be reliable. Techsophy disclaims all warranties as to the accuracy, completeness or adequacy of such information.
          </p>
          <p>
            The Site provides links to web sites and access to content, products and services from third parties, including users, advertisers, affiliates and sponsors of the Site. The User agrees that Techsophy is not responsible for the availability of, and content provided on, third party web sites. The User is requested to peruse the policies posted by other web sites regarding privacy and other topics before use. Techsophy is not responsible for third party content accessible through the Site, including opinions, advice, statements and advertisements, and the User shall bear all risks associated with the use of such content. 
          </p>
          <p>
            Techsophy is not responsible for any loss or damage of any sort that the User may incur from dealing with any third party.
          </p>
          <p>
            Techsophy makes no warranty that: (a) the Site will meet your requirements; (b) the Site will be available on an uninterrupted, timely, secure, or error-free basis; (c) the results that may be obtained from the use of the Site or any services offered through the Site will be accurate or reliable.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="content-body-div"
      style="display: block; padding: 50px 150px;"
    >
      <div
        class="inner-page"
      >
        <p
          style="font-size: 48px; color: rgb(1, 38, 83);"
        >
          Terms of use
        </p>
        <p>
          The term "User" shall mean the User who is browsing the website www.techsophy.com (Site).
        </p>
        <p>
          The domain address www.techsophy.com is owned by Techsophy. All content present on this Site, including all the software, text, images, graphics, video and audio used on this Site, is exclusive and proprietary material owned by Techsophy.
        </p>
        <p>
          The Techsophy name and logo are trademarks which belong to and are owned by Techsophy. These trademarks may not be used in any manner whatsoever without prior written consent from Techsophy.
        </p>
        <p>
          By using the Site, the User agrees to follow and be bound by the following terms and conditions and these terms and conditions may be revised or modified at any time by Techsophy without any notice to the User:
        </p>
        <p>
          No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.
        </p>
        <p>
          No material from this site may be copied, modified, reproduced, republished, uploaded, transmitted, posted or distributed in any form without prior written permission from Techsophy. All rights not expressly granted herein are reserved.
        </p>
        <p>
          Unauthorized use of the materials appearing on this Site may violate copyright, trademark and other applicable laws, and could result in criminal or civil penalties. Techsophy may monitor access to the Site.
        </p>
        <p>
          Techsophy may terminate the User's access to the Site at any time for any reason. The provisions regarding disclaimer of warranty, accuracy of information, and indemnification shall survive such termination.
        </p>
        <p>
          Techsophy makes no warranties, express or implied, including, without limitation, those of merchantability and fitness for a particular purpose, with respect to any information, data, statements or products made available on the Site.
        </p>
        <p>
          The Site, and all content, materials, information, software, products and services provided on the Site, are provided on an "as is" and "as available" basis.
        </p>
        <p>
          Techsophy shall have no responsibility for any damage to User's computer system or loss of data that results from the download of any content, materials, information from the Site.
        </p>
        <p>
          Techsophy may unilaterally change or discontinue any aspect of the Site at any time, including, its content or features.
        </p>
        <p>
          Techsophy will not be liable for damages of any kind, including without limitation, direct, indirect, incidental or consequential damages (including, but not limited to, damages for lost profits, business interruption and loss of programs or information) arising out of the use of or inability to use the Site, or any information provided on the Site, or for any other reason whatsoever.
        </p>
        <p>
          The User agrees to indemnify, defend and hold Techsophy harmless from and against all losses, expenses, damages and costs, including reasonable attorneys' fees, arising out of or relating to any misuse by the User of the content and services provided on the Site.
        </p>
        <p>
          The information contained in the Site has been obtained from sources believed to be reliable. Techsophy disclaims all warranties as to the accuracy, completeness or adequacy of such information.
        </p>
        <p>
          The Site provides links to web sites and access to content, products and services from third parties, including users, advertisers, affiliates and sponsors of the Site. The User agrees that Techsophy is not responsible for the availability of, and content provided on, third party web sites. The User is requested to peruse the policies posted by other web sites regarding privacy and other topics before use. Techsophy is not responsible for third party content accessible through the Site, including opinions, advice, statements and advertisements, and the User shall bear all risks associated with the use of such content. 
        </p>
        <p>
          Techsophy is not responsible for any loss or damage of any sort that the User may incur from dealing with any third party.
        </p>
        <p>
          Techsophy makes no warranty that: (a) the Site will meet your requirements; (b) the Site will be available on an uninterrupted, timely, secure, or error-free basis; (c) the results that may be obtained from the use of the Site or any services offered through the Site will be accurate or reliable.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
